buildSrc/RollupConfig.js

```diff
@@ -153,6 +173,8 @@ export function getChunkName(moduleId, { getModuleInfo }) {
 	if (code.includes("@bundleInto:common-min") || isIn("libs/stream") || isIn("packages/tutanota-utils") || isIn("packages/tutanota-error")) {
 		// if detecting this does not work even though the comment is there, add a blank line after the annotation.
 		return "common-min"
+	} else if (code.includes("@bundleInto:common-functionality")) {
+		return "common-functionality"
```

Comment:
I think we discussed this but I forgot why we need an annotation instead of packing directories in common into it